.github/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# Changes from release 2024/12 to 2024/13
+1. Added D098 complex to sectorfile (Chris Globe)
```

Comment:
```suggestion
1. Added D098 complex - thanks to @RadarController (Chris Globe)
```
Standard CL Format per [the Style Guide](https://github.com/VATSIM-UK/UK-Sector-File/blob/main/.github/Style%20Guide.md#changelog)